src/api.js

```diff
@@ -4,10 +4,12 @@ const baseApi = axios.create({
   baseURL: "https://be-nc-news-oefr.onrender.com/api",
 });
 
-export const getArticles = () => {
-  return baseApi.get("/articles").then(({ data }) => {
-    return data.articles;
-  });
+export const getArticles = (topic) => {
+  if (topic === "articles") {
+    return baseApi.get("/articles").then(({data}) => data.articles)
```

Comment:
Please try to use this function to get all the articles, also to get the ones for a selected topic. You can achieve this with using 'axios params'. See 'Axios/Axios params' in NC notes.